docs/CHANGELOG.rst

```diff
@@ -13,6 +13,7 @@ Unreleased
 Added
 -----
 - Add ``resolwebio/slamdunk`` Docker image
+- Add ``seqtk-rev-complement-single`` process
```

Comment:
Two processes were added, so modify to something like: Add ``seqtk-rev-complement-single`` and ``seqtk-rev-complement-paired`` processes